pkgs/development/compilers/llvm/14/libcxxabi/default.nix

```
@@ -2,9 +2,17 @@
 , monorepoSrc, runCommand
 , cxx-headers, libunwind, version
 , enableShared ? !stdenv.hostPlatform.isStatic
-}:
+}: let
```

Nit I would do
```suggestion
}:

let
```
Using more lines but touching fiewer is good to avoid conflicts down the road.